files/jsondata/L10n-Template.json

```
@@ -72,19 +72,22 @@
     "fr": "Non standard",
     "ja": "非標準",
     "ko": "비표준",
+    "pt-BR": "Não padrão",
     "ru": "Не стандартно",
     "zh-CN": "非标准"
   },
 
   "non_standard_header_copy": {
-    "de": "Diese Funktion ist nicht standardisiert und befindet sich nicht auf dem Weg zur Standardisierung. Verwenden Sie sie nicht auf Produktionsseiten, die dem Web ausgesetzt sind: Sie funktioniert nicht für alle Benutzer. Es kann auch große Inkompatibilitäten zwischen Implementierungen geben, und das Verhalten kann sich in Zukunft ändern.",
-    "en-US": "This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.",
-    "es": "Esta característica no es parte de los estándares. No la uses en sitios Web en producción: no funcionará para todos los usuarios. Podrían haber también incompatibilidades considerables entre distintas implementaciones y el comportamiento podría cambiar en el futuro.",
-    "fr": "Cette fonctionnalité n'est ni standard, ni en voie de standardisation. Ne l'utilisez pas pour des sites accessibles sur le Web : elle ne fonctionnera pas pour tout utilisateur. Il peut également y avoir d'importantes incompatibilités entre les implémentations et son comportement peut être modifié dans le futur.",
-    "ja": "この機能は標準ではなく、標準化の予定もありません。公開されているウェブサイトには使用しないでください。ユーザーによっては使用できないことがあります。実装ごとに大きな差があることもあり、将来は振る舞いが変わるかもしれません。",
-    "ko": "이 기능은 비표준이므로 실제 프로덕션에서 사용하지 마세요. 모든 사용자 환경에서 작동하지 않을 수 도 있으며, 미래에 호환성 문제가 생길 수 있습니다.",
-    "ru": "Эта возможность не является стандартной и стандартизировать её пока никто не собирается. Не используйте её на сайтах, смотрящих во внешний мир: она будет работать не у всех пользователей. Также могут присутствовать большие несовместимости между реализациями и её поведение может в будущем измениться.",
-    "zh-CN": "该特性是非标准的，请尽量不要在生产环境中使用它！"
+    "de": "Wir raten davon ab, diese Funktion auf produktiven Webseiten zu verwenden, da sie nicht bei jedem funktionieren wird.",
+    "en-US": "We do not recommend using this feature on production sites facing the web because it will not work for everyone.",
```

FWIW This isn't correct and it misses the point of the original issue - which is that you will use this on production sites if you need to. It also will work for everyone if they happen to be on the same browser.

Maybe:

```suggestion
    "en-US": "This feature is non-standard and is not on a standards track. Standard features are preferred, but this may be a suitable alternative on some browsers.",
```

---

From MDN's authoritative point of view, I think we should _explicitly_ recommend against using non-standard features on production, and explain why. How about:

```suggestion
    "en-US": "We do not recommend using this non-standard feature in production, as it has limited browser support, and may change or be removed.",
```

cc @Rumyra

---

> From MDN's authoritative point of view, I think we should explicitly recommend against using non-standard features on production, and explain why. 
> Agree with @hamishwillee 's suggestion here

I'm not sure which of these came first :-)

@caugner 
1. This misses the point that the issue specifically argues that the "you can't use in production" is overly prescriptive.
   That is why I've suggested that these can be used in production, albeit under circumstances where it makes sense.
   You might close the issue as invalid if the MDN position is always "we don't recommend in production _full stop_".
2. If you do go with something more like yours, then I would make it non-specific to _**this** non-standard feature" - we apply this everywhere - it isn't really "this" case we're worried about, but the general use. Also "Facing the web" is redundant.
3. A possible compromise that could meet both needs:

```suggestion
    "en-US": "We do not generally recommend non-standard features in production, as that have limited browser support, and may change or be removed. They may be a suitable alternative in some browsers.",
```

---

@hamishwillee If we're making the recommendation general, then we should probably start by explaining that this feature is not standardized, or non-standard, or not on a standards track. Also, is the word "generally" really necessary, given that the statement _is_ general? How about this (e.g. adding "However, …"):

```suggestion
    "en-US": "This feature is not standardized. We do not recommend using non-standard features in production, as they have limited browser support, and may change or be removed. However, they can be a suitable alternative in specific cases where no standard option exists.",
```

---

Looks good. I'm committing this and approving so you can merge at will.